<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connections</class>
 <widget class="QDialog" name="Connections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>connections to database</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>249</width>
     <height>200</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="host_name">
        <property name="text">
         <string>host_name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="host_name_2"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>39</number>
      </property>
      <item>
       <widget class="QLabel" name="db_name">
        <property name="text">
         <string>db_name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="db_name_2"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="connection_port">
        <property name="text">
         <string>connection_port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="connection_port_2"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>62</number>
      </property>
      <item>
       <widget class="QLabel" name="login">
        <property name="text">
         <string>login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="login_2"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="leftMargin">
       <number>39</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password"/>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
